okhttp/src/main/java/okhttp3/HttpUrl.kt

```diff
@@ -828,6 +831,9 @@ class HttpUrl internal constructor(builder: Builder) {
       encodedPathSegments.add("") // The default path is '/' which needs a trailing space.
     }
 
+    /**
+     * @param scheme either http or https
```

Comment:
Can you quote these, "http" or "https" ? Thanks!

Reply:
Sure! Quotes have been added